portfolio/src/main/webapp/script.js

```diff
@@ -16,13 +16,50 @@
  * Adds a random greeting to the page.
  */
 function addRandomGreeting() {
-  const greetings =
-      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+    const greetings =
+        ["Hello, you look cute today. Lets be cute together!", "¡Hola Mundo! Soy Dominicano", "你好，世界！<3 ", "Bonjour le monde! Je t'aime "];
 
-  // Pick a random greeting.
-  const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+    // Pick a random greeting.
+    const greeting = greetings[Math.floor(Math.random() * greetings.length)];
+
+    // Add it to the page.
+    const greetingContainer = document.getElementById('greeting-container');
+    greetingContainer.innerText = greeting;
+
+    scareUser();
+    askIfScared();
+    
+}
+
+function askIfScared() {
+    const sacredText = document.getElementById("scaredText");
```

Comment:
scaredText?

Reply:
Is the variable name not specific enough? Should I change it to didIScareYouText?

Reply:
It looks like a typo, sacredText vs scaredText